tools/sof-dump-status.py

```diff
@@ -413,6 +410,7 @@ def dump_dapm(dapm, filter = "all"):
     parser.add_argument('-P', '--fwpath', action='store_true', help='get firmware path according to DMI info')
     parser.add_argument('-S', '--dsp_status', type=int, help='get current dsp power status, should specify sof card number')
     parser.add_argument('-d', '--dapm', choices=['all', 'on', 'off', 'standby'], help='get current dapm status, this option need root permission to access debugfs')
+    parser.add_argument('-e', '--export', action='store_true', help='export pipeline parameters from proc file system')
```

Comment:
you change this two patches later, can we squash the changes in one location to avoid this tick-tock effect?